### ISSUE TITLE
Client downloads and distributes files from muted spammers

### ISSUE THREAD

**Issue by saber28** [HelloZeroNet/ZeroNet#948]
Can we stop client distributing files from muted spammers?

**Comment by grez911** [comment]
No, because the file distributing based on peers addresses which is IP or onion addresses. Muting based on `auth_address`. And we can not ask peers about their `auth_address` for privacy reasons.
@HelloZeroNet Say, if I am wrong.

**Comment by mkg20001** [comment]
@grez911 It is possible. The path of the file simply needs to be ignored and the file deleted.
